app/mailers/user_emailer.rb

```diff
@@ -0,0 +1,11 @@
+class UserEmailer < ApplicationMailer
+  default from: 'laurawhalin@gmail.com'
```

Comment:
Awesome we get the emails sent from Laura's account. At some point we need the eventual owner of this site to give us an email and credentials.